### PR TITLE
39 header navigation with search bar

### LINKED ISSUE #39: Header Navigation
In Yeti 1.5, we could copy the header/navigation as you have it now, but three main differences:
 
Hide the My Account/Login and Country selection for now, but these can be enabled when we have the code
The search bar, if you click on it then it will go directly to the search results page
Upon clicking on the options, the main navigation would be a left bar, rather than a mega menu.  The reason for suggesting this is reducing development time,  and not recreating the menu from the POC in Yeti 1.5, whilst it might be reviewed again for Yeti 2.0
 
I have attached two screen shots from the mock-ups.  I’m happy to discuss further.
 
 image
 
 
image

### PR DESCRIPTION
**Pull request by Rajesh-GSPANN**
Hi,
Can you please review the changes and approve.

Fix #39 

Test URLs:
- Before: [Stories](https://main--danaher-abcam--aemsites.hlx.page/en-us/stories/articles/attitude-is-everything)
- After: [Stories](https://39-header-navigation--danaher-abcam--aemsites.hlx.page/en-us/stories/articles/attitude-is-everything)

### CODE REVIEW

File: blocks/header/header.js [hunk 1]
```diff
@@ -98,46 +116,46 @@ function buildNavBlock(headerBlock) {
...
-  const exploreFlyout = a({ id: 'explore-flyout', class: 'flex items-center gap-x-1 group', href: '/' }, 'Explore all', span({ class: 'icon icon-chevron-right-orange w-4 h-4 transition-transform group-hover:-translate-x-0.5' }));
-
   const navigateActions = div(
     { class: 'flex justify-between mt-5 text-base text-white font-bold mx-2' },
     backFlyout,
-    exploreFlyout,
   );
 
-  decorateIcons(closeFlyout);
+  // decorateIcons(closeFlyout);
```
Comment (rgravitvl) [inline blocks/header/header.js:157]: remove commented lines

File: blocks/header/header.js [hunk 2]
```diff
@@ -152,9 +170,10 @@ function buildFlyoutMenus(headerBlock) {
           ...(anchorHref && { 'data-href': anchorHref }),
         },
       );
-      if (headerBlock.querySelector('span.icon')) {
+      if (flyMenuChild.querySelector('a') === null) {
```
Comment (rgravitvl) [inline blocks/header/header.js:173]: No need of '=== null', instead you only cam have if (flyMenuChild.querySelector('a'))

File: blocks/header/header.js [hunk 3]
```diff
@@ -20,33 +20,50 @@ function sortFlyoutMenus(menuPath) {
...
-        exploreFlyout.classList.remove('hidden');
-      } else exploreFlyout.classList.add('hidden');
     }
   });
 }
 
 function buildSearchBlock(headerBlock) {
   const searchHtmlBlock = headerBlock.children[0];
-  searchHtmlBlock.classList.add(...'navbar-wrapper justify-center bg-black z-50 pt-2 lg:pt-4'.split(' '));
+  searchHtmlBlock.classList.add(...'navbar-wrapper justify-center bg-black z-50 pt-4'.split(' '));
```
Comment (rgravitvl) [inline blocks/header/header.js:35]: Use applyClasses from script JS

**Review by pardeepgera23 — APPROVED** [review]

File: blocks/header/header.js (hunk 1 above)
Comment (Rajesh-GSPANN) [reply blocks/header/header.js]: Hi @rgravitvl ,
Can you please check now I have removed the comment line

File: blocks/header/header.js (hunk 2 above)
Comment (Rajesh-GSPANN) [reply blocks/header/header.js]: @rgravitvl , Can you please check now.

File: blocks/header/header.js (hunk 3 above)
Comment (Rajesh-GSPANN) [reply blocks/header/header.js]: Updated. Can you please check now @rgravitvl

**Pull request merged**